pages/SavingsForm.js

```diff
@@ -0,0 +1,47 @@
+import AccountForm from './AccountForm';
+
+class SavingsForm extends AccountForm {
+
+    get savingsSavingsButton (){return $('label[for="Savings"]')}
+
+    get moneyMarketSavingsButton (){return $('label[for="Money Market"]')}
+  
+    get savingsNameInput (){return $('#name')}
+  
+    get newSavingsFormSubmitButton () {return $('#newSavingsSubmit')}
+
+    get newSavingsFormResetButton () {return $('#newSavingsReset')}
+
+    get newSavingsErrorText () {return $('#new-account-error-msg')}
```

Comment:
Same comment. This is a shared element between both accounts, therefore AccountForm is a better candidate for placing it (so you don't have to duplicate it)